cmd/conduit/root/initialize/init.go

```diff
@@ -61,10 +61,9 @@ func (c *InitCommand) Docs() ecdysis.Docs {
 func (c *InitCommand) createDirs() error {
 	// These could be used based on the root flags if those were global
 	dirs := []string{"processors", "connectors", "pipelines"}
-	conduitPath := filepath.Dir(c.flags.Path)
```

Comment:
c.flags.Path should be the path to conduit.yaml right @raulb ? That's why we're taking the directory from that path.

Reply:
@hariso actually, no. This change is correct. We have `config.path` which is global and indicates the full path of the configuration file (including `conduit.yaml`). This one in `init` indicates the root directory where the whole conduit structure would be created.